src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -14630,10 +14611,19 @@ void VRFCondenserEquipment::VRFOU_CalcCompH(
                     NumIteCcap = NumIteCcap + 1;
                     goto Label19;
                 }
-                if (CapDiff > (Tolerance * Cap_Eva0)) NumIteCcap = 999;
-
-                Ncomp = this->RatedCompPower * CurveValue(state, this->OUCoolingPWRFT(CounterCompSpdTemp), T_discharge, T_suction);
+                if (CapDiff > (Tolerance * Cap_Eva0)) {
+                    NumIteCcap = 999;
+                    CyclingRatio = min(1.0, (TU_load + Pipe_Q) * C_cap_operation / Cap_Eva1);
+                } else {
+                    CyclingRatio = 1.0;
```

Comment:
This looks odd. This appears to say if the calculation converged set CyclingRatio = 1 ?

Reply:
when it converges, the capacity `Cap_Eva1` and demand `Cap_Eva0` are close to each other, the system should run full-time, thus the cycling ratio should be 1.

Reply:
If `(TU_load + Pipe_Q) * C_cap_operation = Cap_Eva1` when this converges then I agree. I just can't follow the math well enough to see that clearly.

Before it converges:

    CyclingRatio = min(1.0, (TU_load + Pipe_Q) * C_cap_operation / Cap_Eva1);

and after it converges:

    CyclingRatio = 1.;

Then why can't the more detailed calculation happen all the time?

    if (CapDiff > (Tolerance * Cap_Eva0)) NumIteCcap = 999;
    CyclingRatio = min(1.0, (TU_load + Pipe_Q) * C_cap_operation / Cap_Eva1);

Reply:
I might have missed a "-Ncomp", it should be TU_load + Pipe_Q - Ncomp. TU_load + Pipe_Q - Ncomp and Q_evap_req should be equal.